src/cluster/multi_step_cluster.h

```diff
@@ -66,17 +67,28 @@ class MultiStep : public ClusteringAlgorithm {
 };
 
 struct Neighbors : public vector<vector<int>>, public Consumer {
-	Neighbors(size_t n) : vector<vector<int>>(n), smallest_index(n,0), number_edges(n,0){
+	Neighbors(size_t n) : vector<vector<int>>(n), smallest_index(n,0), number_edges(n,0) {
 		iota(smallest_index.begin(), smallest_index.end(), 0);
 	}
 
 	vector<uint32_t> smallest_index;
 	vector<size_t> number_edges;
-	
-	
+	vector<TempFile*> tempfiles;
+	size_t size;
+
+
 	virtual void consume(const char* ptr, size_t n) override {
```

Comment:
`(*this)[query].push_back(subject);`

Soll nur dann passieren, wenn `--external` nicht gesetzt ist.